analyzer/runtime/evm/client.go

```diff
@@ -250,7 +250,7 @@ func EVMDownloadNewNFT(ctx context.Context, logger *log.Logger, source nodeapi.R
 		return nftData, nil
 
 	default:
-		return nil, fmt.Errorf("download stale nft type %v not handled", tokenType)
+		return &EVMNFTData{}, nil
```

Comment:
anachronistically, with this we start the "download functions always return non-nil" (other than when there's an error) convention